lib/hackney/income/less_dangerous_tenancy_gateway.rb

```diff
@@ -104,6 +104,19 @@ def get_tenancy(tenancy_ref:)
         tenancy_item
       end
 
+      def update_tenancy(tenancy_ref:, is_paused_until:)
+        uri = URI("#{@api_host}/tenancies/#{ERB::Util.url_encode(tenancy_ref)}")
```

Comment:
Would like to use this method so that duplcate `/` on api_host are removed

```suggestion
        uri = URI.parse(File.join(@api_host, "/tenancies/#{ERB::Util.url_encode(tenancy_ref)}"))
```

Reply:
Is it safe to assume a file path and a rest endpoint are interchangeable?

Reply:
This seems to be a common usage in rails

Reply:
🚢